### ISSUE TITLE
Need some help with xfce4 installation

### ISSUE THREAD

**Issue by MedoX71T** [BlackArch/blackarch-config-xfce#3]
i installed the FULL ISO and installed xfce4 & xfce4-goodies using 
`pacman -S xfce4 xfce4-goodies`

now i have error because `gst-plugins-base-libs` is old and will break the system so i hit this
`pacman -S xfce4 xfce4-goodies --ignore gst-plugins-base-libs`

and it asked me to skip `parole` package because i ignored gst-plugins-base-libs [y/N]: y

now everything seems OK but when i get login using XFCE i type my username & password it kicks me out why ?
i tried to install `blackarch-config-xfce`  but nothing works everytime i login using XFCE I'm forced to logout automatically

I just want the FULL ISO but with XFCE from the slim iso can you help ?

**Comment by noraj** [comment]
Hi,

1. Please **do not** use the full ISO, it's only meant for offline use like some air gap use cases. It's hardly possible to update it due to conflicting dependencies. Prefer the slim or netinstall ones.
2. What do you mean `gst-plugins-base-libs` is old, [AL package](https://archlinux.org/packages/extra/x86_64/gst-plugins-base-libs/) is sync with last upstream version.
3. I don't think xfce login issues are related to the parole / gst-plugins-base-libs issues.
4. `blackarch-config-xfce` brings some BA theming on XFCE, so if your XFCE doesn't work in the first place this won't make it work any better.

**Comment by MedoX71T** [comment]
thank you for the quick response 
1- i installed the slim iso and its so nice to work with but when i install `blackarch-group` ex: scanner i have a lot of errors like 
`xxxxx package corrupt` with delete question so i decided to install the full iso 

2- what is the proper way to install xfce4 on the full iso and how to debug it ?
like is there a tutorial of wiki with xfce4 login issues i can look at ?

here is the terminal log when i hit startxfce4 
`/sbin/startxfce4: X server already running on display :0.0`
`<stdin>:16:99: warning: missing terminating " character`
`<stdin>:76:99: warning: missing terminating " character`
`xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`
`xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`
`xfce4-session: /usr/lib/libc.so.6: version 'GLIBC_2.38' not found (required by /usr/lib/libpolkit-gobject-1.so.0)`

i already have `glibc` installed because `pacman` works great so how `xfce4-session` can't find `glibc` ?

**Comment by noraj** [comment]
> 1- i installed the slim iso and its so nice to work with but when i install `blackarch-group` ex: scanner i have a lot of errors like
> `xxxxx package corrupt` with delete question so i decided to install the full iso

It's because there are too many tools in the groups and if one of them is broken it will make the whole operation fails. Forget groups, on slim you have a minimal set of tools already install + you can install all the tools you want manually. Anyway groups are useless, is a group contains 500 tools you will never have a use for all of them, just wasting storage and update bandwidth. Just install manually what you need.
The full ISO has the same issues as groups but worth.

> 2- what is the proper way to install xfce4 on the full iso and how to debug it ?

As I said, the full ISO is not meant to be updated or modified, it's just made to be deployed and used as is. Often on air gap environment where it's impossible to install stuff over the network. So there is no easy way to install anything on it without having to deal with troublesome dependency issues.

**TL;DR**: install netinstall or slim ISO + install manually only the tools you need with `pacman -S`. Forget groups and full ISO.

**Comment by noraj** [comment]
For the record, a warning is on the website telling people not to use the full ISO unless you are an expert willing to spend ton of times debbuging:

![image](https://github.com/user-attachments/assets/83dd1222-bf94-46eb-bd91-dc9cb67a26fc)
